FlowCrypt/Controllers/Setup/Helpers/SetupViewController+Setup.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SetupViewController+Setup.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 14.11.2019.
+//  Copyright © 2019 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+
+extension SetupViewController {
+   enum Parts: Int, CaseIterable {
+       case title, description, passPhrase, divider, action, optionalAction
+   }
+
+   enum SetupAction {
+       case recoverKey, createKey
```

Comment:
Note - I would have expected `importKey` here as well. Maybe it's actually `importOrRecoverKey, createKey` ?

Reply:
I see it now, import key is in `ImportKeyViewController` which is a pity because it makes this a bit confusing. But will do